src/Tests/Microsoft.NET.Build.Tests/GivenThatWeWantToUseAnalyzers.cs

```diff
@@ -23,6 +23,37 @@ public GivenThatWeWantToUseAnalyzers(ITestOutputHelper log) : base(log)
         {
         }
 
+        [Theory]
+        [InlineData("WebApp", "EnableRequestDelegateGenerator", false)]
+        [InlineData("WebApp", "EnableRequestDelegateGenerator", true)]
+        public void It_resolves_requestdelegategenerator_correctly(string testAssetName, string property, bool isEnabled)
+        {
+            var asset = _testAssetsManager
+                .CopyTestAsset(testAssetName, identifier: "C#")
+                .WithSource()
+                .WithProjectChanges(project =>
+                {
+                    var ns = project.Root.Name.Namespace;
+                    project.Root.Add(new XElement(ns + "PropertyGroup", new XElement(property, isEnabled)));
```

Comment:
For this property it may be a good idea to also add a test case for when the property isn't set at all (since that will be the common case, I believe).